indico/modules/events/registration/models/registrations.py

```diff
@@ -249,6 +249,12 @@ class Registration(db.Model):
         default=False
     )
 
+    #: The date/time until which the person can modify their registration
+    exceptional_modification_allowed_end_dt = db.Column(
```

Comment:
I was thinking since there already a field called `modification_end_dt` at the regform level that would create confusion if they have the same name. I used this naming to highlight the fact that it's granted exceptionally.

Reply:
In the end I think that naming is simply `modification_end_dt` like the regform also make sense, so I changed it to `modification_end_dt`